dotnet/samples/Protocol/TestEnvoys/Memmon/GetRuntimeStatsResponsePayloadSerialization.g.cs

```diff
@@ -17,11 +17,11 @@ public partial class GetRuntimeStatsResponsePayload
     {
       ""name"": ""diagnosticResults"",
       ""type"": {
-        ""type"": ""map"",
-        ""values"": {
-          ""type"": ""string""
-        },
-        ""default"": {}
+""type"": ""map"",
```

Comment:
nit: What happened to the indentation here?

Reply:
I'm not entirely sure, but this happened after running code gen so it is fine to be checked in